scripts/test

```diff
@@ -18,8 +18,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
         usage
     else
         ./scripts/lint
-        ./scripts/format
```

Comment:
This seems incorrect in the template - we shouldn't run formatting as part of tests